src/ripple/protocol/impl/STArray.cpp

```diff
@@ -32,13 +32,13 @@ STArray::STArray()
     //v_.reserve(reserveSize);
 }
 
-STArray::STArray (STArray&& other)
+STArray::STArray (STArray&& other) noexcept
     : STBase(other.getFName())
     , v_(std::move(other.v_))
 {
```

Comment:
`CountedObject <STArray>` is not nothrow default constructible, but I believe it could be made so.

Reply:
Fixed in 209c9fd7ad71ce26b868c1df276ff184f89f6660